### DISCUSSION TITLE
Lot of feature requests

### DISCUSSION THREAD

**Discussion by NikunjK8** [T8RIN/ImageToolbox#935] [Q&A]
I want to suggest a lot of things (mainly from my stock gallery) & some changes. Should i open seperate issues for them (it'll pile up the issue section a bit but they'll be easy to track seperatly) or mention them in a single issue with checkmarks?

**Comment by NikunjK8** [comment]
Also thank you for the amazing app

**Comment by T8RIN** [comment]
I think it's better to create separate ones

**Reply by NikunjK8** [reply]
Alright. I'll do it after some time. They'll be all optional ofc. I don't want to pressurize you for work. Feel free to close anything you disagree with.

**Reply by T8RIN** [reply]
Thanks! I always like feature requests, because they make the app better
